pkg/dbt_cloud/project.go

```diff
@@ -16,6 +16,8 @@ type Project struct {
 	RepositoryID           *int    `json:"repository_id,integer,omitempty"`
 	State                  int     `json:"state"`
 	AccountID              int     `json:"account_id"`
+	FreshnessJobId         *int    `json:"freshness_job_id"`
```

Comment:
can these be sent to the create/ update endpoints? i.e. are they just extra fields for the project endpoint or is it necessary to manage as a project_artefacts resource at all?

Reply:
Ahh I see, the same reason resources [like this](https://registry.terraform.io/providers/GtheSheep/dbt-cloud/latest/docs/resources/dbt_cloud_project_connection) exist... ok, makes sense, let's do it like this for now so that it's at least possible 😸